lib/gnat.ex

```diff
@@ -90,10 +91,29 @@ defmodule Gnat do
   """
   def unsub(pid, sid, opts \\ []), do: GenServer.call(pid, {:unsub, sid, opts})
 
+  @doc """
+  Ping the NATS server
+
+  This correlates to the [PING](http://nats.io/documentation/internals/nats-protocol/#PINGPONG) command in the NATS protocol.
+  If the NATS server responds with a PONG message this function will return `:ok`
+  ```
+  {:ok, gnat} = Gnat.start_link()
+  :ok = Gnat.ping(gnat)
+  ```
+  """
+  def ping(pid) do
```

Comment:
This is actually pretty common on nats clients, but the method is called `flush`: https://github.com/nats-io/go-nats/blob/master/nats.go#L2464-L2468